hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -1113,17 +1113,22 @@ def test_basic_indices_replaces_whole_axis_slices_with_ellipsis(idx):
 
 
 @given(
-    shape=nps.array_shapes(min_dims=0, max_side=4)
+    nps.array_shapes(min_dims=0, max_side=4)
```

Comment:
Keyword arguments for clarity, please - it's much easier to read the code when the name is right there, instead of needing to check the corresponding index of the signature 🙂